app/routes/circulars/circulars.lib.ts

```diff
@@ -76,7 +76,9 @@ export function subjectIsValid(subject: string) {
   if (subject.length)
     return (
       !emailIsAutoReply(subject) &&
-      validSubjectKeywords.some((x) => subject.startsWith(x))
+      validSubjectKeywords.some((x) =>
+        subject.toLowerCase().startsWith(x.toLowerCase())
+      )
```

Comment:
@dakota002, would you please pull out this change separately? It's OK to push this fix directly to main, if you like.

Reply:
Yup, got it!